### PR TITLE
improvement(KtPopover): Align with Design when Using Options

### PR DESCRIPTION
**Pull request by FlorianWendelborn**
## Before

image

## After

image

Figma (also linked in documentation of course): <https://www.figma.com/file/0yFVivSWXgFf2ddEF92zkf/Kotti-Design-System?node-id=95%3A552>

### CODE REVIEW

File: packages/kotti-ui/source/kotti-popover/KtPopover.vue [hunk 1]
```diff
@@ -63,7 +63,11 @@ export default {
 	},
 	computed: {
 		popperClass() {
-			return this.size ? `kt-popper kt-popper--${this.size}` : `kt-popper`
+			const classes = ['kt-popper', `kt-popper-${this.size}`]
```
Comment (carsoli) [inline packages/kotti-ui/source/kotti-popover/KtPopover.vue:66]: why single-hyphen the size; it's a modification

Comment (FlorianWendelborn) [reply packages/kotti-ui/source/kotti-popover/KtPopover.vue:66]: It’s a typo

Comment (carsoli) [reply packages/kotti-ui/source/kotti-popover/KtPopover.vue]: css also needs editing then

Comment (FlorianWendelborn) [reply packages/kotti-ui/source/kotti-popover/KtPopover.vue]: @carsoli CSS is fine as far as I can tell

Comment (carsoli) [reply packages/kotti-ui/source/kotti-popover/KtPopover.vue]: ok so ur classes are not on this pattern `kt-popper--size-sm`

Comment (carsoli) [reply packages/kotti-ui/source/kotti-popover/KtPopover.vue]: so just missing the `--size` here

Comment (FlorianWendelborn) [reply packages/kotti-ui/source/kotti-popover/KtPopover.vue]: @carsoli Are you absolutely sure you saw the nested `&--size`?

Comment (FlorianWendelborn) [reply packages/kotti-ui/source/kotti-popover/KtPopover.vue]: Okay, I see why you’re confused now I think, the CSS is fine the code still isn’t

**Review by carsoli — APPROVED** [review]

**Pull request merged**